### PR TITLE
new errors page

### PR DESCRIPTION
**Pull request by tgberkeley**

### CODE REVIEW

**Comment by masenf** [comment]
image

is it supposed to look like this? that text needs more contrast

To a broader point though, i don't really love the format of the page here; it would be much faster and easier to update if these errors were in flexdown blocks IMO.

File: pcweb/pages/errors.py [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+                    rx.tbody(
+                        rx.tr(
+                            rx.td("foo"),
+                        ),
+                    )
+                )
+                ```
+                """
+            ),
+            rx.text(f"See upstream docs: {rx.link('here.', href='https://nextjs.org/docs/messages/react-hydration-error')}"),
```
Comment (masenf) [inline pcweb/pages/errors.py:81]: image

something is wrong here

File: pcweb/pages/errors.py [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+                """
+                ),
+            rx.markdown(
+                    r"""
+                ```python
+                rx.table(
+                    rx.tr(
+                        rx.td("foo"),
+                    )
+                )
```
Comment (masenf) [inline pcweb/pages/errors.py:54]: this is from the old chakra table and won't actually work now, we should update the code to use `rx.table.root`, `rx.table.row` and `rx.table.cell`, etc

**Review by masenf — CHANGES_REQUESTED** [review]

**Comment by tgberkeley** [comment]
Change whole page into flexdown to make it easier to edit

File: pcweb/pages/errors.py (hunk 1 above)
Comment (tgberkeley) [reply pcweb/pages/errors.py:81]: fixed this

File: pcweb/pages/errors.py (hunk 2 above)
Comment (tgberkeley) [reply pcweb/pages/errors.py:54]: changed

File: errors.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+                rx.text.strong("No ❌"),
+                rx.code_block(
+"""rx.text(rx.text("foo"))
+rx.text(rx.box("foo"))
+rx.text(rx.center("foo"))""",
+                    language="python",
+                ),
+                rx.code_block(
+"""rx.table.root(
+rx.table.row(
```
Comment (picklelo) [inline errors.md:79]: This should be indented (we can fix in a follow up)

**Review by picklelo — APPROVED** [review]

**Pull request merged**